### PR TITLE
#161782719: setup houndCI

### PR DESCRIPTION
**Pull request by Laludztee**
#### What does this PR do?

Setup HoundCI to perform a style check on PRs

#### How should this be manually tested?

>- Clone the repository
>- Codebase must follow the style guide before merging occurs

#### Relevant pivotal tracker stories

#161782719

### CODE REVIEW

File: .hound.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+fail_on_violations: true
```
Comment (IyiKuyoro) [inline .hound.yml]: Please remove this. It would cause us serious issues.

File: .hound.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+
+eslint:
+  version: 5.8.0
```
Comment (IyiKuyoro) [inline .hound.yml]: This should look more like this
```
eslint:
  enabled: true
  config_file: .eslintrc.json
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+
+[![Build Status](https://img.shields.io/badge/passed-protected__by%3A%20hound-purple.svg?branch=develop)](https://img.shields.io/badge/passed-protected__by%3A%20hound-purple)
```
Comment (IyiKuyoro) [inline README.md]: So we don't need the "passed" in the badge. The badge should look like this.
image

Comment (Laludztee) [reply README.md]: Okay. Gotcha

File: .hound.yml (hunk 2 above)
Comment (Laludztee) [reply .hound.yml]: Okay

File: .hound.yml (hunk 1 above)
Comment (Laludztee) [reply .hound.yml]: Oh. How so?

Comment (IyiKuyoro) [reply .hound.yml]: The build would not work on every mistake you make with styling

Comment (Laludztee) [reply .hound.yml]: Alright

**Pull request merged**